From 230ac7c6c4efc520ece819a913e78a8bd09d2c76 Mon Sep 17 00:00:00 2001
From: Takuro Ashie <ashie@clear-code.com>
Date: Tue, 15 Mar 2022 18:00:47 +0900
Subject: [PATCH 09/10] Remove bashism from wrapper script

Signed-off-by: Takuro Ashie <ashie@clear-code.com>
---
 chrome/installer/linux/common/wrapper | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/chrome/installer/linux/common/wrapper b/chrome/installer/linux/common/wrapper
index 03c7ac4a77154..1b401eabd03b0 100755
--- a/chrome/installer/linux/common/wrapper
+++ b/chrome/installer/linux/common/wrapper
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # Copyright (c) 2011 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
@@ -39,13 +39,7 @@ export CHROME_VERSION_EXTRA="@@CHANNEL@@"
 # We don't want bug-buddy intercepting our crashes. http://crbug.com/24120
 export GNOME_DISABLE_CRASH_DIALOG=SET_BY_GOOGLE_CHROME
 
-# Sanitize std{in,out,err} because they'll be shared with untrusted child
-# processes (http://crbug.com/376567).
-exec < /dev/null
-exec > >(exec cat)
-exec 2> >(exec cat >&2)
-
 CHROME_EXTRA_ARGS=""
 
 # Note: exec -a below is a bashism.
-exec -a "$0" "$HERE/@@PROGNAME@@" ${CHROME_EXTRA_ARGS} "$@"
+exec "$HERE/@@PROGNAME@@" ${CHROME_EXTRA_ARGS} "$@"
-- 
2.34.1

